noxfile.py

```diff
@@ -111,9 +118,12 @@ def dependency_versions_to_be_tested(
         )
     )
 
+    if len(supported_versions) == 0:
+        return []
+
     if len(supported_versions) == 1:
         # Only one version? We surely want to test it!
-        return supported_versions
+        return [supported_versions[0].version]
```

Comment:
this is a bit unclear, is it returning the python version or the version to be tested?

Reply:
i've added a comment, should be clearer now